examples/with-reasonml/pages/_app.js

```diff
@@ -0,0 +1,28 @@
+import App, { Container } from 'next/app'
```

Comment:
This was the file I was trying to avoid adding.

Reply:
If you play around with next for any considerable amount of time you will probably use this file at some point. I don't see a reason to avoid it when it's provided to allow you to do this very thing.